.github/pull_request_template.md

```diff
@@ -1,29 +1,29 @@
 ## Proposed changes
 
-Describe the big picture of your changes here to communicate to the maintainers why the pull request is necessary and why we should accept it. If it fixes a previously reported bug, be sure to link to that issue. Please also summarize the changes to the code.
+Your contribution to PorePy is greatly appreciated. Give the maintainers a clear explanation of why your pull request (PR) is needed and why we should accept it. If this PR is solving an issue, explain how it is done. Please, also summarise the changes to the code. 
 
 ## Types of changes
 
 What types of changes does your code introduce to PorePy?
 _Put an `x` in the boxes that apply_
 
+- [ ] Minor change (e.g., dependency bumps, broken links, etc.)
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Testing (contribution related mainly to testing of existing functionality)
-- [ ] Documentation update (if none of the other choices apply)
+- [ ] Testing (contribution related to testing of existing or new functionality)
+- [ ] Documentation (contribution related to adding, improving, or fixing documentation)
+- [ ] GitHub actions workflows (e.g., workflow files `*.yml`)
 - [ ] Other: 
 
 ## Checklist
 
-_Enter one &check; (`&check;`) in each row. The table can be updated after creating the PR._
+_Put an `x` in the boxes that apply or explain briefly why the box is not relevant_
 
-||Yes|No|NA|
-|---|---|---|---|
-|The update is covered by the  test suite |_|_|_|
-|The documentation is up-to-date |_|_|_|
-|I have not duplicated existing functionality |_|_|_|
+- [ ] The documentation is up-to-date.	
+- [ ] Static typing is included in the update.
+- [ ] I have not duplicated existing functionality.
```

Comment:
As several persons may work on a PR, I would try to circumvent using 'I' here, and maybe circumvent personal pronomina in general. Instead one could say that 'This PR does not ...'.